tuned/profiles/functions/function_intel_recommended_pstate.py

```diff
@@ -0,0 +1,38 @@
+from . import base
+
+processor_name = ("sandybridge",
+        "ivybridge",
+        "haswell",
+        "broadwell",
+        "skylake")
+
+pmu_path = "/sys/devices/cpu/caps/pmu_name"
+active = "active"
+disable = "disable"
+
+class intel_recommended_pstate(base.Function):
+    """
+    Checks the processor code name and return the pstate
```

Comment:
change pstate value to:
recommended intel_pstate CPUFreq driver mode

Reply:
Done